src/cpu/core_dyn_x86/cache.h

```diff
@@ -160,8 +162,10 @@ class CodePageHandler : public PageHandler {
 		addr&=4095;
 		if (host_readd(hostmem+addr)==(Bit32u)val) return;
 		host_writed(hostmem+addr,val);
-		if (!*(Bit32u*)&write_map[addr]) {
```

Comment:
PPC: Fix needs to be aligned and native endian (not swapped to little-endian like the `host_*` functions do).